modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -2710,10 +2742,51 @@ public boolean isAutoActivationEnabled() {
      * @param autoActivation Auto activation enabled flag value.
      * @return {@code this} instance.
      * @see #isAutoActivationEnabled()
+     * @deprecated Use {@link IgniteConfiguration#setClusterStateOnStart(ClusterState)} instead.
      */
+    @Deprecated
     public IgniteConfiguration setAutoActivationEnabled(boolean autoActivation) {
+        U.warn(log, "Property autoActivation deprecated. Use clusterStateOnStart instead.");
```

Comment:
the same

Reply:
See my comment above

Reply:
See my comment in a previous conversation.